actionpack/lib/action_dispatch/testing/integration.rb

```diff
@@ -432,6 +432,8 @@ def method_missing(method, *args, &block)
           super
         end
       end
+
```

Comment:
Can you ✂️ a blank line?
We use no blank line style for `ruby2_keywords`.

https://github.com/rails/rails/blob/031763ab5c199121a0cb814649543cce9af53ea8/railties/lib/rails/configuration.rb#L59-L72